app/web/src/features/auth/section/Section.tsx

```diff
@@ -0,0 +1,29 @@
+import { Typography } from "@material-ui/core";
+import classNames from "classnames";
+import React from "react";
+
+import makeStyles from "../../../utils/makeStyles";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    margin: theme.spacing(4, 0),
+  },
+}));
+
+type SectionProps = {
+  title: string;
+  content: string | React.ReactNode;
```

Comment:
`ReactNode` can already be a string and import `ReactNode` directly instead of `React`
```suggestion
  content: ReactNode;
```

Reply:
Changed in last push